docs/user-guide/cli-install-configure-zosmf-security.md

```diff
@@ -0,0 +1,59 @@
+# Configuring z/OSMF Security
+
+Review the tasks that security administrators must complete to configure z/OSMF security for your Zowe CLI implementation.
+
+:::info **Required role: security administrator**
+:::
+
+## Configuring z/OS REST services SAF security
+
+A security administrator must configure security to allow z/OSMF System Authorization Facility (SAF) access to the resources that Zowe CLI requires. Zowe CLI **[correct?]** uses REST endpoints that are associated with each z/OSMF REST API. After you complete all z/OSMF and z/OSMF cloud provisioning configurations, you can test your connection to z/OSMF to verify that Zowe CLI **[correct?]** can communicate with z/OS systems.
```

Comment:
@adam-wolfe @zFernand0: I took this content from the Brightside doc. No surprisingly, in the Brightside doc, "Zowe CLI" was actually "Brightside." In this file, I replaced all mentions of "Brightside" with "Zowe CLI." But I'd like to confirm that the substitutions are one-for-one. 
Is it correct to reference "Zowe CLI" in these two instances?

Reply:
This is correct.

Reply:
➕ 1️⃣ 
Seems right to me 😋